cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -183,16 +181,16 @@
 {% from 'molecules/notification.html' import render as notification with context %}
 {% import 'organisms/post-preview.html' as post_preview with context %}
 
-{% macro render(controls, index) %}
+{% macro render(controls) %}
     <div class="o-filterable-list-controls"
-         id="o-filterable-list-controls-{{ index | string }}">
+         id="o-filterable-list-controls">
         {% set form = filter_data.forms.pop(0) %}
```

Comment:
I'm not sure if this necessary anymore.

Reply:
@sebworks Are you saying, you're not sure if it's safe to remove?

Reply:
@richaagarwal is there a reason to use `pop`?

Reply:
Spoke with @richaagarwal and code is good.